src/bms.lisp

```diff
@@ -0,0 +1,49 @@
+(defun bms-loop () {
+    (var bms-vmin-limit-start (conf-get 'bms-vmin-limit-start))
+    (var bms-vmax-limit-start (conf-get 'bms-vmax-limit-start))
+    (var bms-tmax-limit-start (- (conf-get 'bms-t-limit-start) 3)) ; Start 3 degrees before Motor CFG -> BMS limiting functionality would happen.
+    (var bms-tmin-limit-start 0)
+    (var bms-cell-balance-start 0.5)
+    (var bms-ic-tmax-limit-start (+ bms-tmax-limit-start 15)) ; Set bms temp limit to +15C past cell limit
```

Comment:
Could you name the variables (on lines 2-7) in a better way? the `bms-` prefix is redundant in this context. The `-start` sufix as well. What about something like:
```
v-cell-max-limit
v-cell-min-limit
temp-cell-max-limit
...
```

Reply:
yeah that works